contributing_to_docs/doc_guidelines.adoc

```diff
@@ -444,12 +444,13 @@ $ lorem.sh
 Without the line spaces, the content is likely to be not parsed correctly.
 
 * It is recommended to include source tags for the programming language used in the code block to enable syntax highlighting. For example, use the source tags
- `[source, bash]`, `[source, yaml]`, `[source, javascript]`.
+ `[source, yaml]` or `[source, javascript]`.
++
+NOTE: Do not use `[source, bash]` for `oc` commands or any terminal commands, as it does not render properly in some cases, so we leave off for consistency across the docs.
```

Comment:
s/as/because
s/cases, so we leave off for consistency across the docs./cases. We will not accept new contributions that contain \`[source, bash]\`.

Reply:
You'll also need to move this into 4.0. (When 4.0 GAs, we're going to deprecate this branch.) For that collection, I'd leave off the last sentence.

Reply:
@kalexand-rh I had opened https://github.com/openshift/openshift-docs/pull/13972 for 4.0, and @vikram-redhat asked me to also include this guidance for 3.x (this PR). I'll update w/ your suggestions here.

And no, I'm not going through and removing all source,bash instances from the 3.x doc set.